zuul.d/projects.yaml

```diff
@@ -171,3 +171,10 @@
 ##            branches: master
 #        - openshift-origin-functional-test-orange:
 #            branches: master
+
+- project:
+    name: apache/spark
+    periodic:
+      jobs:
+        - spark-integration-test-kubeadm-k8s
+        - spark-integration-test-minikube-k8s
```

Comment:
please add `branches: master` for these periodic jobs to avoid launching for each branches.

Reply:
OK, but actually, it won't, only the `master` branch is protected branch, and I have configured `exclude-unprotected-branches` option

Reply:
these two jobs are applied on project `apache/spark`, that's maintained by other community, so they might change branch policy, so adding `branches: master` into OpenLab side make sense